charts/velero/README.md

```diff
@@ -90,6 +90,10 @@ helm upgrade vmware-tanzu/velero <RELEASE NAME> --reuse-values --set configurati
 
 ## Upgrading
 
+### Upgrading to 5.1.1
+
+This version removes the `nodeAgent.privileged` field, you should use `nodeAgent.containerSecurityContext.privileged` instead
+
```

Comment:
This is the note for velero major/minor version upgrade doc.
Please consider have a comment in values.yaml instead.

Reply:
Generally, we made backward compatible for each helm chart minor/patch version.
If it can't be backward compatible, we bump the helm chart major version.

Reply:
> Yes, I understand, we're doing the same for our charts, but where do you document breaking changes or little migration steps like this?

Sorry, no. We don't have the doc sections mentioned the steps for the breaking change.
If you like, we could create a new doc section for the breaking change one.
But for this PR case, I think it should able to make the helm chart backward compatible.

Reply:
Yeah, I was thinking about the "major break"ability of this change as well...

The current implementation, just removing the field, isn't really backwards compatible.

For one there is no schema, so the user doesn't even get a validation error, and on the otherside the field is just silently ignored as if it would still work.

So this should either be a breaking change or I'd have to merge the privileged field.

Third possibility would be to add a validation to disallow this field in lieu of implementing the whole schema for this little change. But I would kinda classify this as a breaking change as well.

Fourth solution would be to just silently drop it but also not mark it as a breaking change, which wouldn't be very nice to the users tho 😅

What do you think?

Reply:
Or let's keep it as is since the purpose it to deduplicate the duplicated fields. WDYT?

Reply:
> Or let's keep it as is since the purpose it to deduplicate the duplicated fields. WDYT?

Yeah, then I'll just change this to a major release as well. See the README on how I thought we could add info about this 👌